plugins/vpc-bridge/config/k8s_connector.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Don't you need to delete the `.refactor` file as well?

Reply:
done, removed this file